integration-fe/src/components/TableInstance/TableInstance.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component, PropTypes } from 'react';
+import DataRow from './DataRow';
+import '../../styles/main.less';
+import './styles/styles.less';
+
+class TableInstance extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      instances: [],
+    };
+    this.onConfigureNew = this.onConfigureNew.bind(this);
+  }
+
+  componentWillMount() {
+    this.setState({
+      instances: this.props.instanceList,
+    });
+  }
+
+  componentDidMount() {
+    this.props.fetchUserId();
+    this.props.fetchInstanceList();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.instanceList !== nextProps.instanceList) {
+      this.setState({
+        instances: nextProps.instanceList,
+      });
+    }
+  }
+
+  onConfigureNew(e) {
+    e.preventDefault();
+  }
+
+  render() {
+    let dataRowObj = {};
+    return (
+      <div className='wrapper'>
+        <header>
```

Comment:
This component should be just an table component, shouldn't it? I could import just the table and use other header component. What do you think?